packages/mobile/.gitignore

```diff
@@ -88,6 +88,6 @@ android/app/src/main/res/mipmap*
 ios/celo/Base.lproj/
 ios/celo/Images.xcassets/
 src/brandingConfig.ts
-src/icons/Logo.v2.tsx
+src/icons/Logo.tsx
```

Comment:
@jeanregisser is the Logo file auto-generated from the Valora branding repo? If so, how can I change it to make sure it is not generating with ".v2" in the file name?

Reply:
I'm not sure, but I think this may be causing the `yarn.lock` error that the CI test is giving me right now

Reply:
I think it might come from the branding repo: https://github.com/clabs-co/valora-app-branding/tree/master/src/icons

Reply:
Yes you need to update it on the `valora-app-branding` repo and update the sha in https://github.com/celo-org/celo-monorepo/blob/5b949c6dbbe6ce6f5ba9d46ff77609f1db0defeb/packages/mobile/scripts/sync_branding.sh#L24

Reply:
I've gone ahead and done that in anticipation of this PR merging